### PR TITLE
[PDND-38] Repository review for DAF Kafka Consumer

### PR DESCRIPTION
**Pull request by CrisTofani**
Added:

- New Readme file
- Authors file
- issue_template
- License

### CODE REVIEW

**Review by giux78 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,92 @@
...
-kubectl apply -f kubernetes/test/kafka-consumer.yaml
-
-[DEPLOY KAFKA-CONSUMER PROD]
-sudo docker build --no-cache -t nexus.daf.teamdigitale.it/kafka-consumer:1.0.0 .
-sudo docker push nexus.daf.teamdigitale.it/kafka-consumer:1.0.0
-kubectl delete -f kubernetes/prod/kafka-consumer.yaml
-kubectl apply -f kubernetes/prod/kafka-consumer.yaml
\ No newline at end of file
+# DAF Kafka Consumer for Piattaforma Digitale Nazionale Dati (PDND), previously DAF
+<!--> Insert here a brief description of what your repository contains. Insert also links to the last release, the official page, the extended documentation, and other useful external resources.-->
```
Comment (altroisi) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,92 @@
...
+
+## What is the PDND (previously DAF)?
+
+PDND stays for "Piattaforma Digitale Nazionale Dati" (the Italian Digital Data Platform), previously known as Data & Analytics Framework (DAF).
+
+You can find more informations about the PDND on the official [Digital Transformation Team website](https://teamdigitale.governo.it/it/projects/daf.htm).
+
+## What is DAF Kafka Consumer?
+
+<!--> Insert here an extended description of the project with informations about context, goals, stakeholders, use cases, and finally the role of the project within the PDND with links to other repositories requiring this code or this code depends on. Embed also screenshots or video if present to give a preview of the application.-->
```
Comment (altroisi) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,92 @@
...
+
+DAF Kafka Consumer is a backend component based on NodeJS. The aim of this component is to listen 
+the input of a new message in two different kafka topics,*creationfeed* and *notifications*, 
+and process the messages to (in order) create a new dataset and insert a specific notification.
+
+<!--> Insert here informations about files and folders structure, branch model adopted and release policy.-->
```
Comment (altroisi) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,92 @@
...
+
+All the active code is available in the file `serverNew.js` on the **master** branch. To add new feature create a new _feature branch_ from 
+master.
```
Comment (altroisi) [inline README.md]: and _dev_ branch?

File: README.md [hunk 5]
```diff
@@ -1,13 +1,92 @@
...
+
+### Related PDND Services
+
+* **DAF Catalog Manager** available [here](https://github.com/italia/daf-srv-catalog/tree/master)
+* **DAF Security Manager** available [here](https://github.com/italia/daf-srv-security/tree/master)
+* **DAF Dataportal Backend** available [here](https://github.com/italia/daf-dataportal-backend)
+
+## How to install and use DAF Kafka Consumer
+
+> Insert here a brief documentation to use this project as an end-user (not a developer) if applicable, including pre-requisites and internal and external dependencies. Insert a link to an extended documentation (user manual) if present.
```
Comment (altroisi) [inline README.md]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1,13 +1,92 @@
...
+### Run the app
+```
+node serverNew.js
+```
+
+To make the component work you need to have a running Kafka service and configure the correct _Zookeeper_ 
+connection to Kafka
+## How to build and test DAF Kafka Consumer
+
+<!--> Insert here a brief documentation for the developer to build, test and contribute. Insert a link to an extended documentation (developer manual) if present.-->
```
Comment (altroisi) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,13 +1,92 @@
...
+To build the component you can use Docker:
+```
+sudo docker build --no-cache -t <YOUR_DOCKER_IMAGE_NAME> .
+```
+
+## How to contribute
+
+Contributions are welcome. Feel free to [open issues](./issues) and submit a [pull request](./pulls) at any time, but please read [our handbook](https://github.com/teamdigitale/daf-handbook) first.
```
Comment (altroisi) [inline README.md]:
```suggestion
Contributions are welcome. Feel free to [open issues](../../issues) and submit a [pull request](../../pulls) at any time, but please read [our handbook](https://github.com/teamdigitale/daf-handbook) first.
```

**Review by altroisi — CHANGES_REQUESTED** [review]
fix some typos in README.md

File: README.md (hunk 4 above)
Comment (CrisTofani) [reply README.md]: Added the reference to _dev_ branch

**Pull request merged**